src/views/account/Grades/Graph/GradesAverage.tsx

```diff
@@ -90,8 +91,6 @@ const GradesAverageGraph: React.FC<GradesAverageGraphProps> = ({
     let maxAvg = getPronoteAverage(grades, "max");
     let minAvg = getPronoteAverage(grades, "min");
 
```

Comment:
pq pas des constantes ? :)

```suggestion
    const maxAvg = getPronoteAverage(grades, "max");
    const minAvg = getPronoteAverage(grades, "min");

```